util/memory/tracker.go

```diff
@@ -275,3 +278,25 @@ func (t *Tracker) BytesToString(numBytes int64) string {
 
 	return fmt.Sprintf("%v Bytes", numBytes)
 }
+
+// AttachToGlobalTracker attach the tracker to the global tracker
+// AttachToGlobalTracker should be called at the initialization for the session executor's tracker
+func (t *Tracker) AttachToGlobalTracker(globalTracker *Tracker) {
+	if t.parent != nil {
+		t.parent.remove(t)
```

Comment:
Should not call `t.parent.remove` here since the global tracker does not have any children and should not run into mutex area.

Reply:
updated.